### PR TITLE
Quick Fix for arclines... again!

### PR DESCRIPTION
**Pull request by rcooke-ast**
This makes the algorithm a little less conservative in the rejection of patterns.

I was half way through a cross-match code for multislit, which snuck in here, but it doesn't do anything yet. Travis tests pass, and I'm fairly sure this will pass the dev suite, but that takes ~1 hour, so here is the fix in the meantime.

### CODE REVIEW

File: pypeit/core/wavecal/autoid.py [hunk 1]
```diff
@@ -524,6 +528,23 @@ def run(self):
...
+        # First cross correlate all spectra
+        corr = signal.correlate(self._spec[:, 0], self._spec[:, 4], mode='same')
+        amax = np.argmax(corr)
+        print(amax)
+        from matplotlib import pyplot as plt
+        xplt = np.arange(self._spec.shape[0])
+        plt.plot(xplt, self._spec[:, 0], 'k-', drawstyle='steps')
+        plt.plot(xplt+(amax-self._spec.shape[0]//2), self._spec[:, 4], 'r-', drawstyle='steps')
+        plt.show()
+        pdb.set_trace()
```
Comment (profxj) [inline pypeit/core/wavecal/autoid.py]: Lingering pdb

**Comment by rcooke-ast** [comment]
This part of the code wasn't doing anything anyway, so I've removed it, and will reinstate it for a future PR.

**Comment by profxj** [comment]
merge once Travis passes.

**Comment by profxj** [comment]
merging.

**Pull request merged**